db/migrate/20210311152412_add_api_user_to_audit.rb

```diff
@@ -0,0 +1,6 @@
+class AddApiUserToAudit < ActiveRecord::Migration[6.0]
+  def change
+    add_column :audits, :api_user_id, :integer
+    add_index :audits, :api_user_id
```

Comment:
This is absolutely fine, but just as a tip, you can achieve the same thing as these two lines with `add_reference(: audits, :api_user, foreign_key: true)`

Reply:
ah yes, I actually did that last time I added an index - but this time I was lazy and copy pasted it from another one of our files hahahaaa will change it, its the last migration so best be neat